packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -48,21 +54,68 @@ export interface DashboardLayoutSlots {
    * @default Link
    */
   appTitle?: React.ElementType;
+
   /**
    * The toolbar actions component used in the layout header.
    * @default ToolbarActions
+   * @see [DashboardLayout#slots](https://mui.com/toolpad/core/react-dashboard-layout/#slots)
    */
   toolbarActions?: React.JSXElementConstructor<{}>;
+
   /**
-   * The toolbar account component used in the layout header.
+   * The toolbar actions component used in the layout header.
    * @default Account
+   * @deprecated it is moved to the toolbarActions component
+   * @see [DashboardLayout#slots](https://mui.com/toolpad/core/react-dashboard-layout/#slots)
    */
   toolbarAccount?: React.JSXElementConstructor<AccountProps>;
+
   /**
    * Optional footer component used in the layout sidebar.
    * @default null
    */
   sidebarFooter?: React.JSXElementConstructor<SidebarFooterProps>;
+
+  /**
+   * This component will completely override the internal layout of the top bar.
+   * You can still use the built-in `<AppTitle />`, `<ThemeSwitcher />` and `<Account />` components inside.
+   * @default null
+   * @see [DashboardLayout#slots](https://mui.com/toolpad/core/react-dashboard-layout/#slots)
```

Comment:
If we don't have a slot example that shows this slot being used we should not add this @see line.